### ISSUE TITLE
replace last layer

### ISSUE THREAD

**Issue by qnkhuat** [matterport/Mask_RCNN#497]
Hi, I want to change the coco dataset with just 3 classes. 
So:
+ is this ok to use pre-train?
+ if use pre-train can I just place a softmax with 3 class at last?
+ Can anyone give me some hints about how to do change the last layer? 😄 
Thanks a lot.

**Comment by austinmw** [comment]
Follow-up question, if we have 3 classes do we need 3+1=4 to include background?
